Makefile

```diff
@@ -16,8 +16,11 @@ clean: ## remove generated byte code, coverage reports, and build artifacts
 	rm -fr dist/
 	rm -fr *.egg-info
 
-requirements: ## install development environment requirements
+pre-requirements: ## install Python requirements for running pip-tools
```

Comment:
Should we call this `piptools` to try to be in sync with cookiecutter? See https://github.com/openedx/edx-cookiecutters/blob/80bf2b5acf13e0522637fb8897862b431efe0d0a/python-template/%7B%7Bcookiecutter.placeholder_repo_name%7D%7D/Makefile#L60

Reply:
Ah, sure. Some other repos use `pre-requirements` but it looks like `piptools` is actually more common! (So, it's good that that's what the cookiecutter uses.)